docker-entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+mkdir /etc/cups/
+echo "ServerName $CUPSD_SERVER_NAME" > /etc/cups/client.conf
+exec "$@"
```

Comment:
How does this file work with the docker-entrypoint file provided in the psd-deployment? Does one overright the other?

Reply:
Need to add call to rake task too  `bundle exec rake printers:create`